src/components/pages/Authentication/hooks.ts

```diff
@@ -14,23 +14,42 @@ export default () => {
   const navigate = useNavigate();
   const location = useLocation();
   const [currentTeam, setTeam] = useTeam();
+  const [currentWorkspaceId, setCurrentWorkspaceId] = useWorkspaceId();
+  const [currentUserId, setCurrentUserId] = useUserId();
   const [, setNotification] = useNotification();
   const passwordPolicy = window.REEARTH_CONFIG?.passwordPolicy;
   const t = useT();
 
   const { data, loading } = useGetTeamsQuery({ skip: !isAuthenticated });
-  const teamId = currentTeam?.id || data?.me?.myTeam.id;
+
+  if (isAuthenticated && !currentUserId) {
+    setCurrentUserId(data?.me?.id);
+  }
+  let teamId = currentTeam?.id || data?.me?.myTeam.id;
```

Comment:
While you are making changes/updates, if there is a little bit of code you can clean up or fix up please do. We still have some "team" related code in reearth, but long ago we made "team" become "workspace". So this code here would be perfect to update to workspaceId